src/tokens.ts

```diff
@@ -38,19 +38,21 @@ export default class Tokens {
       const ethereumTokens = new Map();
 
       Object.values(ethereumConfig).forEach((contractDecimals, index) => {
-        const contract = contractDecimals.contract;
-        const decimals = contractDecimals.decimals;
+        const address = contractDecimals.contract;
```

Comment:
:wrench: I think it would be clearer to replace `contractDecimals` with `tokenConfig` given the name of the interface (or even just `token`).

Reply:
How about `tokenDetail` instead?